source/01-LE/02.ptx

```diff
@@ -73,35 +73,179 @@
 </definition>
 
 <activity estimated-time='10'>
-    <!-- TODO split into activities per operation, have students perform op on example -->
-    <statement>
+    <introduction>
```

Comment:
Rescaffold opening activity into seven parts for improved Specific Choice / Simultaneous Response.  Include examples of each proposed row op so the proposal is clear to students, and facilitate the explanation why or why not each works.